corehq/apps/userreports/app_manager/helpers.py

```diff
@@ -24,16 +19,26 @@ def get_case_data_sources(app):
 
 
 def get_case_data_source(app, case_type):
-    prop_map = get_case_properties(app, [case_type], defaults=list(DEFAULT_CASE_PROPERTY_DATATYPES))
+    schema = CaseExportDataSchema.generate_schema_from_builds(
+        app.domain,
+        app._id,
+        case_type,
+        only_process_current_builds=True,
+    )
+    # the first two (row number and case id) are redundant/export specific,
+    meta_properties_to_use = MAIN_CASE_TABLE_PROPERTIES[2:]
```

Comment:
would you want to make this a different const?

Reply:
that const would be something like `CASE_TABLE_PROPERTIES_RELEVANT_TO_UCRS` or something? 

the slicing here definitely makes me feel a little icky but am unsure whether it'd be worth it to create that separately. one thing that helps with the ickiness is that tests will break if someone changes that list in a way that breaks this.

happy to do it in a follow up if you think good though. no strong feelings really.

Reply:
:+1: